docs/wiki/GettingStarted.md

```diff
@@ -115,24 +122,22 @@ On of the most important actions you should take from the beginning is to update
 
 Please refer to [this list][AzureNames] to check which services have a global scope and must be updated.
 
-### Dependencies
+### Platform principal
```

Comment:
Maybe add some info about what it is being referred to inside the repository like 'deploymentSpId', so that there is correlation between those two names

Reply:
Added a reference at the end of the section